solid/src/test/java/com/inrupt/client/solid/SolidClientTest.java

```diff
@@ -413,23 +672,58 @@ public int statusCode() {
                         })
         );
         assertEquals(statusCode, exception.getStatusCode());
+        // On malformed response, the ProblemDetails should fall back to defaults.
```

Comment:
Does this mean the client will ignore weird or corrupted responses from the server?

Reply:
That is correct: any response that is not valid Problem Details, either because RFC9457 is not supported by the server at all or because the server returns a malformed response while having the `application/problem+json` `Content-Type` will have the same behavior of creating a default `ProblemDetails` object (only having the status code) that will be tacked on the exception.